src/algorithmForNN/pointCloudUtils.py

```diff
@@ -151,6 +151,111 @@ def get_area_of_plane(points: o3d.geometry.PointCloud, plane_model: list) -> flo
     return current_list[1] * current_list[2]
 
 
+def get_calibration_matrix_from_calib_file(path_to_calibration_file: str) -> np.ndarray:
+    calib_matrix = np.zeros((4, 4))
+    with open(path_to_calibration_file) as file:
+        (
```

Comment:
Oh, looks crazy... We don't pay for number of lines... :)))

Firstly, you can initialize drectly with `calib_matrix = np.eye(4)`

Secondly

`calib_matrix[:3,:4] = np.array(list(map(float, file.readlines()[4][4:].rstrip().split(" ")))).reshape(3, 4)` , check, maybe it is fine even without call of "list" inside.